pipeline/src/collection.py

```diff
@@ -161,11 +175,10 @@ def load(self, *paths, version=DEFAULT_VERSION):
         """
         if len(paths) == 1 and os.path.isdir(paths[0]):
             data_dir = paths[0]
-            json_paths = [
-                os.path.join(data_dir, item)
-                for item in os.listdir(data_dir)
-                if os.path.splitext(item)[1] in (".json", ".jsonld")
-            ]
+            json_paths = (
+                glob(f"{data_dir}/**/*.jsonld", recursive=True, include_hidden=False)
+                + glob(f"{data_dir}/**/*.json", recursive=True, include_hidden=False)
+            )
```

Comment:
```suggestion
            json_paths = (
                glob(f"{data_dir}/**/*.jsonld", recursive=True)
                + glob(f"{data_dir}/**/*.json", recursive=True)
            )
```

Should run on Python 3.9